docs/extensions.md

```diff
@@ -2,18 +2,113 @@
 
 ## Creating extensions
 
-### Extension template
+OCDS extensions follow the structure of the [standard extension template](https://github.com/open-contracting/standard_extension_template). The template copies the structure of the core schema (except for _versioned-release-validation-schema.json_, see below) and allows you to add or update fields in those places where the extension should modify the schema using [JSON merge patch](https://tools.ietf.org/html/rfc7396).
 
-[Extension template](https://github.com/open-contracting/standard_extension_template)
+The core repository for OCDS has 4 [schema files](https://github.com/open-contracting/standard/tree/master/standard/schema):
+
+* release-schema.json
+* release-package-schema.json
+* record-package-schema.json
+* versioned-release-validation-schema.json
+
+_versioned-release-validation-schema.json_ is produced programmaticallly (see [Make validation schema](../deployment/standard-live#make-validation-schema) in the standard deployment section), which means that you don't have to author it manually when creating an extension.
+
+**TODO**: Mechanisn for extension authors to create _versioned-release-validation-schema.json_ in their extensions.
+
+All extensions must have an [extension.json](https://github.com/open-contracting/standard_extension_template/blob/master/extension.json), with `"name"` and `"description"` fields required.
```

Comment:
The [official documentation](http://standard.open-contracting.org/latest/en/extensions/developing/) points to the template repository's readme, and a lot of the documentation here repeats it. I think it's best to centralize all common "how to author an extension" information in that repo, and just link to it.

I think we can move the naming, description and codelist guidance there, as it's common to all extensions.

Similarly, for the other extension tools, I think we can just use the handbook as a quick reference to which tools exist, each with a one-line description. The detail for extension_registry is already (mostly?) in its readme, so we should just update that readme if any important detail is missing.